### PR TITLE
Fix Edge audio settings modal overflow / toast issues

### LINKED ISSUE #6794: Change audio settings modal overflows in Edge
When I say "No" to the echo test and am sent to the modal to change my audio settings it looks like the following in Edge. Firefox and Chrome don't seem to have any issues.

image

### PR DESCRIPTION
**Pull request by KDSBrowne**
Closes #6794, #6997 and #5798

### CODE REVIEW

File: bigbluebutton-html5/imports/ui/components/toast/component.jsx [hunk 1]
```diff
@@ -23,38 +23,91 @@ const defaultIcons = {
...
+    }
+  }
+  return strings;
+};
+
+const formatMessageStrings = (messages) => {
+  const _messages = messages;
+  const MAX_MESSAGE_CHARS = 40;
+  _messages.forEach((item, index) => {
+    if (_messages[index].length > MAX_MESSAGE_CHARS) {
```
Comment (antobinary) [inline bigbluebutton-html5/imports/ui/components/toast/component.jsx]: Isn't `_messages[index]` the same as `item` ?

Comment (capilkey) [reply bigbluebutton-html5/imports/ui/components/toast/component.jsx]: The whole forEach() should probably just be a map() and don't use the index at all.

**Comment by capilkey** [comment]
I just pull the branch and tested it and saw this for the toast messages,
image

**Comment by capilkey** [comment]
The look of the toast messages for chat are a little weird now. There's a big white space and the text is prematurely cut off. 

New:
image

Old:
image

It looks like there's still a column for the old "..." indicator even though there's nothing displayed there anymore. It should be changed to make the text fill the full toast width.

**Comment by capilkey** [comment]
The toast placement and modal overflow in Edge both look good. Just the toast width needs work.

**Comment by capilkey** [comment]
The toasts still don't look right.

image

image

**Comment by KDSBrowne** [comment]
The behaviour in Firefox seems to be different, I'll fix it to reflect what it looks like in chrome.

image

**Comment by capilkey** [comment]
Strange. The first image that I posted was in Chrome and the second was in Firefox. I don't know why my toast message is getting cut off early while yours is not.

**Pull request merged**